src/components/_cards/BondingTableCard.tsx

```diff
@@ -227,61 +232,66 @@ const BondingTableCard: VFC<TableProps> = (props) => {
             </Tr>
           </Thead>
           <Tbody fontWeight="bold">
-            {userStakes?.length &&
-              userStakes.map((data, i) => {
-                const { amount, lock, rewards, unbondTimestamp } =
-                  data
-                const lockMap: { [key: string]: string } = {
-                  "0": "7 days",
-                  "1": "14 days",
-                  "2": "21 days",
-                }
-                // const unbondTime = new Date(
-                //   unbondTimestamp * 1000
-                // ).toLocaleDateString()
+            {outputUserData.data.userStake?.userStakes?.length &&
```

Comment:
Not a blocker, but may be worth destructuring some of this:

```ts
const { data } = outputUserData || {}
const { userStake } = data || {}

...
```

May reduce some of the drilling here